test/unit/gateways/stripe_test.rb

```diff
@@ -12,6 +12,10 @@ def setup
 
     @options = {
       :billing_address => address(),
+      :shipping_address => address(),
+      :description => 'Test Purchase',
+      :carrier => 'UPS',
+      :tracking_number => '12345'
```

Comment:
@whitby3001 This line is missing a trailing comma, causing the test suite to fail. If you still want this, would you mind patching and rebasing on `master`? If so, I can merge this today or tomorrow.

Reply:
Will do!

Reply:
Thanks!